cli/cmd/start/start.go

```diff
@@ -5,10 +5,12 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
 	"path/filepath"
 
 	"github.com/mattn/go-colorable"
 	"github.com/rilldata/rill/cli/pkg/browser"
+	config2 "github.com/rilldata/rill/cli/pkg/config"
```

Comment:
No need to alias this, can just be `config`

Reply:
It is because there was a local variable config, editor auto imported like this. Have renamed the local variable to `conf`